lib/rubocop/cop/style/mutable_constant.rb

```diff
@@ -14,6 +14,11 @@ module Style
       # positives. Luckily, there is no harm in freezing an already
       # frozen object.
       #
+      # From Ruby 3.0, this cop honours the magic comment
```

Comment:
Please, also add a couple of code snippets with `shareable_constant_value` in the cop examples.